package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "splash-screen",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "bootstrap": "^5.2.1",
+    "node-sass": "^7.0.3",
```

Comment:
what is this used for?

Reply:
I thought it was used for the bootstrap, but I tested and it's working fine without it. Removed.